data/json/itemgroups/magazines.json

```diff
@@ -48,10 +48,10 @@
       ["m1911mag", 10],
       ["m1918mag", 5],
       ["m9mag", 10],
-      ["stenmag", 40],
       ["ppsh35mag", 15]
```

Comment:
I did not add the sten mags to surplus. They were there before I created the mod.